scripts/kernel-crawler.py

```diff
@@ -244,18 +244,14 @@
     })
 repos['AmazonLinux'] = amazon_repos
 
-prev_months = 24
-now = time.localtime()
-check_months = [time.localtime(time.mktime((now.tm_year, now.tm_mon - n, 1, 0, 0, 0, 0, 0, 0)))[:2] for n in range(prev_months)]
 amazon_linux2 = []
-for year, month in check_months[:-1]:
-    amazon_linux2.append({
-        "root": "http://amazonlinux.us-east-1.amazonaws.com/" + str(year) + "." + str(month).zfill(2) + "/core/latest/x86_64/mirror.list",
-        "discovery_pattern": "SELECT * FROM packages WHERE name LIKE 'kernel%'",
-        "subdirs": [""],
-        "page_pattern": "",
-        "exclude_patterns": ["doc", "tools", "headers"]
-        })
+amazon_linux2.append({
+    "root": "http://amazonlinux.us-east-1.amazonaws.com/2/core/2.0/x86_64/mirror.list",
```

Comment:
I tried replacing the `2.0` string with `latest` and it also worked. Maybe you could add both?
http://amazonlinux.us-east-1.amazonaws.com/2/core/2.0/x86_64/mirror.list
http://amazonlinux.us-east-1.amazonaws.com/2/core/latest/x86_64/mirror.list

Reply:
yep, good idea. Fixed.